lib/admin_plugins/roles.js

```diff
@@ -35,7 +33,7 @@ roles.actions = [{
 
 function createOrSaveRole (role, client, callback) {
 
-  var method = _.isEmpty(role._id) ? 'POST' : 'PUT';
+  var method = !role._id ? 'POST' : 'PUT';
```

Comment:
Not sure matters in this case, but isEmpty() is safe in case role is null, but the latter will throw due to accessing a property of a null object

Reply:
Good catch! Made these changes in commit 4387bcedd2a5c220427963e68fec856f55d62188